src/fides/api/service/connectors/query_config.py

```diff
@@ -51,6 +51,16 @@ class QueryConfig(Generic[T], ABC):
     def __init__(self, node: ExecutionNode):
         self.node = node
 
+    @property
+    def partitioning(self) -> Optional[Dict]:  # pylint: disable=R1711
+        # decided to de-scope partitioning support to only bigquery as this grew more complex,
+        # but keeping more generic support stubbed out feels like a reasonable step.
+        if self.node.collection.partitioning:
+            logger.warning(
+                "Partitioning is only supported on BigQuery connectors at this time!"
+            )
```

Comment:
this seems useful, I could have added that for bigquery row-level deletes as well